src/utils/EPToolkit.ts

```diff
@@ -79,6 +79,12 @@ export function exchange_text(text: string, options: IOptions): Buffer {
   let bytes = new BufferHelper();
   bytes.concat(init_printer_bytes);
   bytes.concat(default_space_bytes);
+
+  // utf8 bytes
+  if(options.encoding ==='UTF8'){
+      bytes.concat(Buffer.from([0x1C, 0x26, 0x1C, 0x43, 0xFF]));
```

Comment:
could you use the `options_controller` variable and define `Buffer.from([0x1C, 0x26, 0x1C, 0x43, 0xFF])` as the way we did with `cut` / `beep` or `tailingLines`?

Reply:
Hello @HeligPfleigh, my bad, didn't mean to include this change in this pull request, but once you pointed it out, I'll update using the same coding style and maybe we can merge this as well

p.s In this fix we have set the printer encoding to correctly print unicode characters, if you don't think this is a proper fix, I can remove it from this PR so we could integrate only the build.gradle update to unblock the android build.

Reply:
yes, I think we could include the encoding option into this PR too, just need to follow the previous coding style.

Reply:
I've refactored a little bit the above code, but there are a few mentions

I've added an array with most common encodings but there is a little problem, in the ESC/POS standard actually there is no support for Unicode.

In order to do so, some printers (like [SUNMI](https://file.cdn.sunmi.com/SUNMIDOCS/SunmiPrinter-Developer-Docs-1-1.pdf)) actually do support UTF8 by sending the multiple byte encoding command `0x1C 0x26` (in ESC/POS standard this is mainly used to support multiple byte Chinese characters) and then the UTF8 command `0x1C 0x43 0xFF`.

```
const encoding_mappings_bytes = {
  // single byte encodings
  "CP437": Buffer.from([27, 116, 0]),
  // multiple bit encodings
  "GB18030": Buffer.from([28, 38, 28, 67, 0]),
  "BIG5": Buffer.from([28, 38, 28, 67, 1]),
  "UTF8": Buffer.from([28, 38, 28, 67, 255]),
};
```

This being said, if there aren't any other implications maybe a better idea would be to use the `CP437` as the default value in `default_options` variable since not all of them do support `UTF8`?